src/main/webapp/WEB-INF/redirects.properties

```diff
@@ -12,30 +12,51 @@
 /community/=/support/
 /community=/support/
 
-# Blogs
+# Blogs redirects. NOTE: These will move to the blogs repo in the future for customization by the blogs team.
+/news/=/blog/
 /news/*=/blog/
 /blog/2017/11/29/liberty-spring-boot.html=/guides/spring-boot.html
 /blog/2019/03/01/microprofile-concurrency.html=/blog/2019/08/16/microprofile-context-propagation.html
 /blog/2019/05/24/testing-database-connections-REST-APIs.html=/blog/2019/09/13/testing-database-connections-REST-APIs.html
 /blog/2019/09/13/microprofile-reactive-messsaging-19009.html=/blog/2019/09/13/microprofile-reactive-messaging-19009.html
 
-# Docs
+# Docs redirects. NOTE: These will move to the docs repo in the future for customization by the docs team.
 /config/rwlp_config_*=/docs/ref/config/#
+/config/=/docs/ref/config/
 /config/*=/docs/ref/config/#
+
+# Redirects for a file name change to feature-overview.html. 20.0.0.8 was the latest version with the old file name.
+/docs/ref/feature/featureOverview.html=/docs/latest/feature/feature-overview.html
+/docs/20.0.0.7/reference/feature/featureOverview.html=/docs/20.0.0.7/reference/feature/feature-overview.html
+/docs/20.0.0.8/reference/feature/featureOverview.html=/docs/20.0.0.8/reference/feature/feature-overview.html
+
+# Redirects for a file name change to server-configuration-overview.html. 20.0.0.8 was the latest version with the old file name.
+/docs/ref/config/serverConfiguration.html=/docs/latest/config/server-configuration-overview.html
+/docs/20.0.0.7/reference/config/serverConfiguration.html=/docs/20.0.0.7/reference/config/server-configuration-overview.html
+/docs/20.0.0.8/reference/config/serverConfiguration.html=/docs/20.0.0.8/reference/config/server-configuration-overview.html
+
+# Redirect the docs link in the header to the latest version
 /docs/=/docs/latest/
+
+# Redirect 'latest' docs to the latest doc version. Latest is used in the URL by the Doc macros at https://github.com/OpenLiberty/docs/blob/vNext/lib/ol-asciidoc.js. Latest is also used in links to always link to the most recent doc version even when new versions are released.
 /docs/latest/*=/docs/20.0.0.8/
+/docs/latest/=/docs/20.0.0.8/
+
+# Redirect the pre-Antora docs URLS to the new URL format post-Antora
+/docs/ref/general/*=/docs/latest/reference/
+/docs/ref/config/*=/docs/latest/reference/config/
+/docs/ref/command/*=/docs/latest/reference/command/
+/docs/ref/feature/*=/docs/latest/reference/feature/
```

Comment:
Lets add another comment after these along the lines of:
"Note that we are purposefully not redirecting the above URLs if they're just ending with "/" (aka nothing for the wildcard to match) because that's handled client-side in order to redirect our old hash based URLs via JS."